xclim/__init__.py

```diff
@@ -27,10 +35,10 @@ def build_module(name, objs, doc="", source=None, mode="ignore"):
       keyed by the name they will take in the created module.
     doc : str
       Docstring of the new module.
-    source : Module object
+    source : object
       Module where objects are defined if not explicitly given.
-    mode : {'raise', 'warn', 'ignore'}
-      How to deal with missing objects.
+    mode : str
```

Comment:
Numpy docstring convention: 
```
When a parameter can only assume one of a fixed set of values, those values can be listed in braces, with the default appearing first:

order : {'C', 'F', 'A'}
    Description of `order`.
```

Reply:
Had no idea that that was possible. Sorry for that!